nevergrad/functions/test_base.py

```diff
@@ -23,7 +23,18 @@ def test_experiment_function() -> None:
         constkwarg="blublu",
         plop=p.Choice([3, 4]),
     ))
+    iarrayfunc = base.ArrayExperimentFunction(lambda x: sum(x),   # type: ignore
+        p.Array(shape=(10,)).set_bounds(-0.5, 6.),
+        symmetry=247,
+    )
+    iarrayfunc2 = base.ArrayExperimentFunction(lambda x: sum(x),   # type: ignore
+        p.Array(shape=(10,)).set_bounds(-0.5, 6.),
+        symmetry=111,
+    )
     np.testing.assert_equal(ifunc.dimension, 8)
+    np.testing.assert_equal(iarrayfunc.dimension, 10)
+    assert iarrayfunc(np.zeros(10)) == iarrayfunc.copy()(np.zeros(10))
+    assert iarrayfunc(np.ones(10)) != iarrayfunc2(np.ones(10))
```

Comment:
please put this is a dedicated test, tests should be unitary as much as possible.

Reply:
Also, probably test against an actual value that you *expect*, to make sure it works.

Reply:
> please put this is a dedicated test, tests should be unitary as much as possible.

Done.

Reply:
> Also, probably test against an actual value that you _expect_, to make sure it works.

Done.